.vib/sonarqube/goss/goss.yaml

```diff
@@ -0,0 +1,25 @@
+file:
+  /bitnami/sonarqube:
+    exists: true
+    filetype: directory
+    mode: "2775"
+    owner: root
+  /opt/bitnami/sonarqube/conf/sonar.properties:
+    exists: true
+    filetype: file
+    contains:
+     - sonar.web.port={{ .Vars.containerPorts.http }}
+     - sonar.search.port={{ .Vars.containerPorts.elastic }}
+     - sonarqube-postgresql:{{ .Vars.postgresql.service.ports.postgresql }}/{{ .Vars.postgresql.auth.database }}
```

Comment:
You should also check both `postgresql.auth.username` and `postgresql.auth.password`.